docs/contributor/unsorted/simplestreams-metadata.md

```diff
@@ -89,8 +86,7 @@ after the stream to which they belong.
 The index file must be called "index.(s)json" (sjson for signed). The product file and
 agent binary tarball name(s) match whatever is in the index/product files.
 
-Configuration
-=============
+## Configuration
 
```

Comment:
Fix needs to be applied to `Agent Binary Metadata Contents` above :point_up:

Reply:
Fixed.